src/components/Layout/TableOfContents.jsx

```diff
@@ -16,24 +16,25 @@ const Links = ({ entries }) => (
   </StyledLinkList>
 )
 
-const ChapterList = ({ chapters, entries, title, level = 0 }) => (
+const ChapterList = ({chapters, entries, title, level = 0}) => (
   <StyledChapterList>
     {title && <ChapterListItem key={`${title}${level}`}>
       <ChapterTitle level={level}>{title}</ChapterTitle>
     </ChapterListItem>}
     <ChapterListItem>
-      {entries && <Links entries={entries} />}
+      {entries && <Links entries={entries}/>}
```

Comment:
This is even worse because not only will prettier readd this emptyspace but eslint complains about it missing. You should configure your IDE to at least respect the eslint rules defined by the file `.eslintrc.json`. If not the whole eslint setup makes no sense.